util/settings/settings.go

```diff
@@ -127,6 +130,8 @@ const (
 	settingsApplicationInstanceLabelKey = "application.instanceLabelKey"
 	// resourcesCustomizationsKey is the key to the map of resource overrides
 	resourcesCustomizationsKey = "resource.customizations"
+	// templatingToolsKey is the key to the list of custom templating tools
+	templatingToolsKey = "templatingTools"
```

Comment:
`templatingTools` --> `customTemplatingTools`

Reply:
After adding `custom` prefix data structure names are too long. I feel custom is 
little redundant so to be consistent I've removed `custom` prefix at all and now structure names are just `TemplatingTool` and `ApplicationSourceTemplatingTool`

Reply:
Cool. I think it should be consistent, so `TemplatingTool` is a good option.